tests/lib/parse.js

```diff
@@ -33,10 +33,19 @@ function getRaw(ast) {
 
 describe("parse()", function() {
 
+
+    describe("basic functionality", function() {
+
+        it("should parse an empty string", function() {
+            parser.parse("");
```

Comment:
Please add simple assertions for the resulting ASTs - they will be small enough to keep inline. I would prefer to be explicit rather than the successful test simply be the absence of an exception